modules/install-sno-single-node-openshift-requirements.adoc

```diff
@@ -0,0 +1,35 @@
+// This is included in the following assemblies:
+//
+// installing_bare_metal_single_node/install-sno-single-node-ocp-on-bare-metal.adoc
+
+[id="single-node-openshift-requirements_{context}"]
+
+= Single node OpenShift requirements
+
+Single node OpenShift (SNO) alleviates some of the requirements of high availability and large scale clusters. However, you must address the following requirements:
+
+* *Administration node:* You must have an administration node or laptop to access link:https://cloud.redhat.com/openshift/assisted-installer/clusters[Install OpenShift with the Assisted Installer], to create the cluster, to create the USB boot drive, and to monitor the installation.
+
+* *Production-grade server:* A single node cluster requires a server with sufficient resources to run {product-title} services and a production workload.
++
+.Hardware Requirements
+[options="header"]
+|====
+|Profile|CPU|Memory|Storage
+|Minimum|8 CPU cores|32GB of RAM| 120GB
+|====
++
+The server must have a Baseboard Management Controller (BMC) when booting with virtual media.
+
+* *Networking:* The server must have access to the internet, or access to a local registry if it is not connected to a routable network.  The server must have a DHCP reservation or a static IP address for the Kubernetes API, Ingress route, and cluster node domain names. You must configure the DNS to resolve the IP address to each of the following fully qualified domain names (FQDN):
++
+.Network Requirements
+[options="header"]
+|====
+|Usage|FQDN|Description
```

Comment:
If these are DNS requirements instead of network requirements, then I'd use the same headings that are mentioned on the page for the [any platform](https://docs.openshift.com/container-platform/4.7/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-dns-user-infra_installing-platform-agnostic) installation page.